.gitmodules

```diff
@@ -8,5 +8,5 @@
   branch = main
 [submodule "ccpp/physics"]
   path = ccpp/physics
-  url = https://github.com/NOAA-GSL/ccpp-physics
-  branch = gsl/develop
+  url = https://github.com/SamuelTrahanNOAA/ccpp-physics
+  branch = feature/gsl-develop-20211203-radar-tten
```

Comment:
This points to your branch. Is this intended? If Yes, you will change this later, right?

Reply:
Yes. You have to do this until a merge so people can clone your branch from the ufs-weather-model level.